clientconn.go

```diff
@@ -435,6 +464,24 @@ func (cc *ClientConn) lbWatcher() {
 	}
 }
 
+func (cc *ClientConn) scWatcher() {
+	for {
+		select {
+		case sc, ok := <-cc.dopts.sc:
+			if !ok {
+				return
+			}
+			cc.mu.Lock()
+			// TODO: load balance policy runtime change is ignored.
```

Comment:
As the comment suggested, we will do even though I am not convinced we should provide this feature since it is error-prone and not necessary for almost all the users.